modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
This.

We need to be *really* careful about how users are meant to use this.

If we set users up so that they inject the graph into their objects, then we've messed up - the graph isn't supposed to leave main.